tests/src-android/cgeo/geocaching/connector/gc/GCConnectorTest.java

```diff
@@ -36,14 +36,14 @@ public static void testGetViewport() {
                 final Viewport viewport = new Viewport(new Geopoint("N 52° 25.369 E 9° 35.499"), new Geopoint("N 52° 25.600 E 9° 36.200"));
                 final SearchResult searchResult = ConnectorFactory.searchByViewport(viewport, tokens);
                 assertThat(searchResult).isNotNull();
-                assertThat(searchResult.isEmpty()).isFalse();
```

Comment:
I assume the `isEmpty().isFalse()` was intentional here, to check the the result contains caches (general function) but not contains the mentioned geocode. Otherwise it would pass if the result is empty (false positive result). I would therefore leave it in this testcase rather than moving it to the positive case, where the check for the mentioned cache is enough to verify that the result contains caches.